src/utilities.js

```diff
@@ -107,13 +107,34 @@ export function getResults(searchType, obj) {
     },
   };
 
+  // Create array of items to query
+  var comparisonItems = [];
+
+  // If we are requesting an array of keywords for comparison
+  if (typeof obj.keyword === 'object' && obj.keyword.length) {
+    // Map the keywords to the items array
+    obj.keyword.map((keyword) => {
+
+      // Clone the original obj to keep the dates etc
+      let obj2 = JSON.parse(JSON.stringify(obj));
+
+      // Set the new keyword
+      obj2.keyword = keyword;
+
+      // Add the keyword to the array
+      comparisonItems.push(obj2);
```

Comment:
Rather than having a side effect within this `.map`, use a `.reduce` on `obj.keyword` to create a new array populated with the original object and the keywords replaced.  For example:
```js
comparisonItem = obj.keyword.reduce((arr, keyword) => {
 const copiedObj = {...obj, keyword};

 arr.push(copiedObj);
 return arr;
}, [])
```